scripts/disk_check.py

```diff
@@ -132,13 +153,36 @@ def do_mnt(dirs):
     return ret
 
 
-def is_mounted(dirs):
+def do_unmnt(dirs, overlay_prefix):
+    for d in dirs:
+        d_name = get_dname(d)
+
+        ret = run_cmd(["umount", "-l", "{}_{}".format(overlay_prefix, d_name)])
```

Comment:
It's worth noting that if there's some process in `/etc` or `/home` (including any user's shell being in their home directory), then this will most likely fail. I guess since `disk_check.py` gets executed frequently, it'll eventually work, at least.

Reply:
This will not fail because unmount with -l, lazy mode will wait and unmount in background.